legacy_app.py

```
@@ -85,6 +84,7 @@
 from app.services import recipe_store
 from app.services.food_store import get_food
 from app.services.intervention_trigger_engine import build_targets_next_action
+from app.services.scheduler_access import get_update_scheduler as get_update_scheduler
```

Flake8 (7.3.0)

[error] 'app.services.scheduler_access.get_update_scheduler' imported but unused

(F401)